app/jobs/generate_iteration_snippet_job.rb

```diff
@@ -14,10 +14,8 @@ def perform(iteration)
     file = iteration.submission.files.first
     return unless file
 
-    # TODO: (Required) Set this through Exercism config
-    url = "https://internal.exercism.org/extract_snippet"
     snippet = RestClient.post(
-      url,
+      Exercism.config.snippet_generator_url,
```

Comment:
See https://github.com/exercism/terraform/blob/main/terraform/dynamodb_config.tf#L341 for the production key/value pair